DotNet/mapping/types.js

```diff
@@ -41,7 +41,8 @@ const types = {
         name: () => 'SessionResults',
     },
     "TestResults": {
-        name: () => 'TestResults',
+        name: () => `TestResults`,
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9645502)
```suggestion
        name: () => "TestResults",
```